spec/integration/elasticsearch/transform/manifest.spec.yml

```diff
@@ -1,45 +1,61 @@
-              sort:
-                $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/settings/properties/index/properties/sort"
-          number_of_shards:
-            $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/settings/properties/number_of_shards"
-          analysis:
-            $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/settings/properties/analysis"
-      mappings:
-        $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/mappings"
-      ingest_pipeline:
-        $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/ingest_pipeline"
-      data_stream:
-        $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template/properties/data_stream"
-    required: false
-  start:
-    description: Determines if the transform will be started upon installation
-    type: boolean
-    default: true
\ No newline at end of file
+spec:
+  type: object
+  additionalProperties: false
+  properties:
+    destination_index_template:
+      description: Elasticsearch index template for the transform's destination index
+      type: object
+      additionalProperties: false
+      properties:
+        settings:
+          type: object
+          additionalProperties: false
+          properties:
+            analysis:
+              $ref: "../../data_stream/manifest.spec.yml#/definitions/index_template_setting_analysis"
+            index:
+              type: object
+              additionalProperties: false
+              properties:
+                codec:
+                  $ref: "../../data_stream/manifest.spec.yml#/definitions/index_template_setting_index/properties/codec"
+                mapping:
+                  $ref: "../../data_stream/manifest.spec.yml#/definitions/index_template_setting_index/properties/mapping"
+                mode:
+                  description: Index mode for lookup indices.
+                  type: string
+                  enum:
+                    - lookup
+                refresh_interval:
+                  description: >
+                    How often to perform a refresh operation, which makes recent changes to the index visible to search.
+                  type: string
+                  example: 1s
+                sort:
+                  $ref: "../../data_stream/manifest.spec.yml#/definitions/index_template_setting_index/properties/sort"
+                number_of_shards:
+                  $ref: "../../data_stream/manifest.spec.yml#/definitions/index_template_setting_number_of_shards"
+                number_of_routing_shards:
+                  description: Number used when splitting shards.
```

Comment:
Not sure that this `description` field is in the final JSON schema (after merging the `$ref`). But in any case, at least that description could help the developers as it is part of the code.